### PR TITLE
Removes all master authentication cookies from ws-agent request on haproxy

### PR DESCRIPTION
**Pull request by riuvshin**
### What does this PR do?
Removes all master authentication cookies from ws-agent request on haproxy

### What issues does this PR fix or reference?
https://github.com/codenvy/infrastructure/issues/84

#### Changelog
Remove all master authentication cookies from ws-agent request on haproxy

### CODE REVIEW

File: dockerfiles/init/modules/haproxy/templates/haproxy.cfg.erb [hunk 1]
```diff
@@ -35,6 +35,9 @@ backend tomcat_bk
 
 backend nginx_bk
     mode http
+    http-request replace-header Cookie (.*)session-access-key=([^;]*);(.*) \1\3
```
Comment (skryzhny) [inline dockerfiles/init/modules/haproxy/templates/haproxy.cfg.erb]: Probably we need to add case where our cookie is last and don't have trailing ';'

Comment (garagatyi) [inline dockerfiles/init/modules/haproxy/templates/haproxy.cfg.erb]: +1

Comment (skryzhny) [inline dockerfiles/init/modules/haproxy/templates/haproxy.cfg.erb]: Like `other-cookie=rdwdr;session-access-key=mhgh`

Comment (garagatyi) [inline dockerfiles/init/modules/haproxy/templates/haproxy.cfg.erb]: So correct regexp is: `http-request replace-header Cookie (.*)session-access-key=([^;]*);?(.*) \1\3`

**Pull request merged**